app/src/main/java/com/infomaniak/mail/ui/main/thread/calendar/CalendarEventBannerView.kt

```diff
@@ -135,15 +178,24 @@ class CalendarEventBannerView @JvmOverloads constructor(
     fun initCallback(
         navigateToAttendeesBottomSheet: (List<Attendee>) -> Unit,
         navigateToDownloadProgressDialog: () -> Unit,
+        replyToCalendarEvent: (AttendanceState) -> Unit,
+        onAttendeesButtonClicked: (Boolean) -> Unit,
     ) {
         this.navigateToAttendeesBottomSheet = navigateToAttendeesBottomSheet
         this.navigateToDownloadProgressDialog = navigateToDownloadProgressDialog
+        this.replyToCalendarEvent = replyToCalendarEvent
+        this.onAttendeesButtonClicked = onAttendeesButtonClicked
     }
 
-    private fun MaterialButton.handleChoiceButtonBehavior() {
+    private fun MaterialButton.handleChoiceButtonBehavior(attendanceState: AttendanceState) {
         setOnClickListener {
-            val changedSelectedButton = isChecked
-            if (changedSelectedButton) resetChoiceButtons()
+            val previouslyUnchecked = isChecked
+
+            if (previouslyUnchecked) {
```

Comment:
I find this assignement cause more head scratching than just using isChecked directly
I had to understand why would you purposely do that, and still I'm not convince it's useful